felix/aws/ec2.go

```diff
@@ -85,6 +85,7 @@ func WaitForEC2SrcDstCheckUpdate(check string, healthAgg *health.HealthAggregato
 		jitter        = 0.1
 	)
 
+	//nolint:staticcheck // Ignore SA1019 deprecated
```

Comment:
should we have a FIXME comment here pointing out exactly what is deprecated and that we should remove it in the future?

Reply:
Regarding the ignored staticcheck, `wait.NewExponentialBackoffManger` is [deprecated in k8s/apimachinery](https://github.com/kubernetes/apimachinery/blob/194b6f8ddf62f55d1b8d997223e94ace2c1def31/pkg/util/wait/backoff.go#L354) v1.27+. The suggested replacement is `wait.BackoffUntil` but [it still needs](https://github.com/kubernetes/apimachinery/blob/194b6f8ddf62f55d1b8d997223e94ace2c1def31/pkg/util/wait/backoff.go#L211) a `BackoffManager` as the second parameter. We can't really get away from the `NewExponentialBackoffManger` call to instantiate an instance. This issue is tracked internally and we will remove or fix this when the k8s dependency is updated.

Reply:
thanks for clarifying @hjiawei! sounds good to me!